src/docbkx/chapter77-express-data-prep.xml

```diff
@@ -0,0 +1,370 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<chapter xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://www.oasis-open.org/docbook/4.5" xmlns:xl="http://www.w3.org/1999/xlink"
+         xsi:schemaLocation="http://www.oasis-open.org/docbook/4.5 http://www.oasis-open.org/docbook/xsd/4.5/docbook.xsd">
+
+    <title>
+        Express Data Prep Wizard
+        <inlinemediaobject>
+            <imageobject>
+                <imagedata fileref="notice_commercial_editions_only.png"
+                           format="PNG"/>
+            </imageobject>
+        </inlinemediaobject>
+    </title>
+    <chapterinfo>
+        <abstract>
+            <para>
+                Express Data Prep Wizard is a tool available in Commercial Editions that helps
+                customers to process their postal data faster and more efficiently.
+            </para>
+        </abstract>
+    </chapterinfo>
+
+    <section id="edp_wizard_context">
+        <title>Context</title>
+        <para>
+            Use this wizard on yours or your client's customer lists to develop an automated data processing
+            solution to optimize data, improve contact, remove waste as well as (optionally) secure postal
+            discounts for more effective customer communications.
+        </para>
+
+        <mediaobject>
+            <imageobject>
+                <imagedata fileref="edp-wizard-welcome-page.png" format="PNG" scalefit="1"/>
+            </imageobject>
+        </mediaobject>
+
+        <para>
+            This wizard (How can I effectively communicate with customers) walks you through the necessary
```

Comment:
To be able to use the "quotes" I also changed "This" to "The". 

In my opinion the previous sentence was fine but I don't have problem changing it this way.